modules/test/playwright/tests/cookies-banner-web/cookiesBannerScript.spec.ts

```diff
@@ -0,0 +1,224 @@
+
+	await page.getByRole('tab', {name: 'Control Panel'}).click();
+
+	await page.getByRole('menuitem', {name: 'Sites'}).click();
+
+	await page.getByRole('link', {name: 'Add Site'}).click();
+
+	await page
+		.getByRole('button', {name: 'Select Template: Blank Site'})
+		.click();
+
+	const siteName = getRandomString();
+
+	const addSiteName = await page
+		.frameLocator('iframe[title="Add Site"]')
+		.getByLabel('Name\n\n\t\t\t\n\t\t\t\t\n\n\t\t\t\tRequired');
+
+	await addSiteName.waitFor({state: 'visible'});
+
+	await addSiteName.click();
+
+	await addSiteName.fill(siteName);
+
+	await page
+		.frameLocator('iframe[title="Add Site"]')
+		.getByRole('button', {name: 'Add'})
+		.click();
+
+	await page
+		.getByText('Success: Site was successfully added.')
+		.waitFor({state: 'visible'});
+
+	await page
+		.getByRole('heading', {name: 'Site Settings'})
+		.waitFor({state: 'visible'});
+
+	const productMenuButton = await page.getByLabel('Open Product Menu');
+
+	if (await productMenuButton.isVisible()) {
+		await productMenuButton.click();
+	}
+
+	// Create Page
+
+	await page
+		.getByRole('menuitem', {name: 'Site Builder'})
+		.waitFor({state: 'visible'});
+	await page.getByRole('menuitem', {name: 'Site Builder'}).click();
+
+	await page
+		.getByRole('menuitem', {exact: true, name: 'Pages'})
+		.waitFor({state: 'visible'});
+	await page.getByRole('menuitem', {exact: true, name: 'Pages'}).click();
+
+	await page
+		.locator('li')
+		.filter({hasText: 'New'})
+		.getByRole('button')
+		.waitFor({state: 'visible'});
+	await page
+		.locator('li')
+		.filter({hasText: 'New'})
+		.getByRole('button')
+		.click();
+
+	await page
+		.getByRole('menuitem', {exact: true, name: 'Page'})
+		.waitFor({state: 'visible'});
+	await page.getByRole('menuitem', {exact: true, name: 'Page'}).click();
+
+	await page.getByRole('button', {name: 'Blank'}).waitFor({state: 'visible'});
+	await page.getByRole('button', {name: 'Blank'}).click();
+
+	const addPage = await page.frameLocator('iframe[title="Add Page"]');
+
+	await addPage.getByPlaceholder('Add Page Name').waitFor({state: 'visible'});
+	await addPage.getByPlaceholder('Add Page Name').click();
+
+	const pageName = getRandomString();
+
+	await addPage.getByPlaceholder('Add Page Name').fill(pageName);
+
+	await addPage.getByRole('button', {name: 'Add'}).click();
+
+	// Add HTML Fragment and content
+
+	const searchFragment = await page.getByPlaceholder('Search...');
+
+	await searchFragment.waitFor({state: 'visible'});
+	await searchFragment.click();
+	await searchFragment.fill('html');
+
+	const htmlItem = page.getByRole('menuitem', {
+		name: 'HTML Add HTML Mark HTML as Favorite',
+	});
+
+	await htmlItem.waitFor({state: 'visible'});
+
+	await htmlItem.dragTo(page.locator('#page-editor div').nth(1));
+
+	const htmlExample = await page.getByText('HTML Example');
+
+	await htmlExample.waitFor({state: 'visible'});
+	await htmlExample.click();
+	await htmlExample.click();
+	await htmlExample.click();
```

Comment:
Is triple click intentional?

Reply:
Yes, was the only way I found to edit html fragment.